go.mod

```diff
@@ -9,36 +9,36 @@ require (
 	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
 	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
-	k8s.io/api v0.17.9
-	k8s.io/apimachinery v0.17.9
-	k8s.io/client-go v0.17.9
-	k8s.io/cluster-bootstrap v0.17.9
+	k8s.io/api v0.17.5
```

Comment:
Shouldnl't it be 1.17.4? Or am I missing something?

Reply:
This is just the revert, on k8s 1.17.4 we already had this mismatch, I don't know why. I also noticed it when I upgraded it to 1.17.9

Reply:
wow! then all good